src/Banking/Statement.php

```diff
@@ -35,6 +41,15 @@ public function getOldBalance(): ?Balance
         return $this->oldBalance;
     }
 
+    public function getOldBalanceOrThrowException(): Balance
```

Comment:
Do you really need this method ? I think we could do this in the original getter method and just add a `has<Old|New>Balance` instead.

WDYT?

Reply:
done